dev/com.ibm.ws.security.social/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -331,7 +331,12 @@
         <AD id="authorizationEndpoint" name="%authorizationEndpoint" description="%authorizationEndpoint.desc"
             required="false" type="String" />
         <AD id="tokenEndpoint" name="%tokenEndpoint" description="%tokenEndpoint.desc"
+            required="false" type="String" />
+        <!--  userInfoEndpoint is not required, but if used successfully, results are available in an api.  -->
+        <AD id="userInfoEndpoint" name="%userInfoEndpoint" description="%userInfoEndpoint.desc"
```

Comment:
I think it's better practice to leave new configuration options as internal until we have adequate FATs in place that can verify the attribute behaves as expected.

Reply:
you are right, and I should have said the fat's are coming on #13378, but this one has to go first.